hi-ml-histopathology/src/histopathology/datasets/panda_tiles_dataset.py

```diff
@@ -41,14 +41,21 @@ def __init__(self,
         super().__init__(root=Path(root) / self._RELATIVE_ROOT_FOLDER,
                          dataset_csv=dataset_csv,
                          dataset_df=dataset_df,
-                         train=None)
+                         train=None,
+                         validate_columns=False)
         if occupancy_threshold is not None:
             self.dataset_df: pd.DataFrame
             dataset_df_filtered = self.dataset_df.loc[
                 self.dataset_df['occupancy'] > occupancy_threshold
             ]
             self.dataset_df = dataset_df_filtered
 
+        # Change "left" --> "tile_x" and "top" --> "tile_y"
+        # to be consistent with TilesDataset `TILE_X_COLUMN` and `TILE_Y_COLUMN`
+        self.dataset_df.rename(columns={"left": TilesDataset.TILE_X_COLUMN, "top": TilesDataset.TILE_Y_COLUMN},
```

Comment:
It means that the column names in the dataframe were just renamed to be consistent with the parent class `TilesDataset`, because output handler uses the column names from `TilesDataset`. The columns will be there in output CSV with name `tile_x` and `tile_y`. Ok, I will be working on these outputs in another PR

Reply:
Thanks!